geomstats/geometry/hyperbolic.py

```diff
@@ -5,11 +5,10 @@
 
 
 import geomstats.backend as gs
-from geomstats.geometry.manifold import Manifold
 from geomstats.geometry.riemannian_metric import RiemannianMetric
 
 
-class Hyperbolic(Manifold):
+class Hyperbolic:
```

Comment:
Why doesn't ```Hyperbolic``` inherit from ```Manifold``` anymore ?

Reply:
This class doesnt actually implement any manifold, but wraps the different representations of hyperbolic spaces (`Hyperboloid`, `PoincareBall`, `PoincareHalfSpace`), I need to clarify however how it should interact with these. What I've done so far works to instantiate one of `Hyperboloid`, `PoincareBall`, `PoincareHalfSpace` but `Hyperbolic(2)` does not work.

Reply:
Ok I see. Then ```Hyperbolic``` is another abstract class (like ```Manifold```, ```EmbeddedManifold``` etc.) that cannot be instantiated ?